src/main/java/org/example/learningsystem/core/destination/dto/AccessTokenResponseDto.java

```diff
@@ -0,0 +1,11 @@
+package org.example.learningsystem.core.destination.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record AccessTokenResponseDto(
+        @JsonProperty("access_token") String accessToken,
+        @JsonProperty("token_type") String tokenType,
+        @JsonProperty("expires_in") Integer expiresIn,
```

Comment:
Instead of manually setting name for each field, you can put on class `@JsonNaming(SnakeCaseStrategy.class)`, it does the same